dask_ml/_compat.py

```diff
@@ -29,6 +29,14 @@
 PANDAS_1_2_0 = PANDAS_VERSION > packaging.version.parse("1.2.0")
 WINDOWS = os.name == "nt"
 
+SKLEARN_1_1_X = SK_VERSION >= packaging.version.parse("1.1")
```

Comment:
This name is misleading, since it is `True` for 1.2 (say). I realise this mimics the naming above, so I guess fine.

Reply:
This is the convention used by packages like `six`.